lib/scholar/metrics.ex

```diff
@@ -29,34 +54,65 @@ defmodule Scholar.Metrics do
 
   ## Examples
 
-      iex> Scholar.Metrics.accuracy(Nx.tensor([[0, 1], [1, 0], [1, 0]]), Nx.tensor([[0, 1], [1, 0], [0, 1]]))
+      iex> Scholar.Metrics.onehot_accuracy(Nx.tensor([[0, 1], [1, 0], [1, 0]]), Nx.tensor([[0, 1], [1, 0], [0, 1]]))
       #Nx.Tensor<
         f32
         0.6666666865348816
       >
+      iex> y_true = Nx.tensor([[1, 0, 0, 0], [0, 1, 0, 0], [0, 0, 1, 0], [0, 0, 0, 1]])
+      iex> y_pred = Nx.tensor([[1, 0, 0, 0], [0, 0, 1, 0], [0, 1, 0, 0], [0, 0, 0, 1]])
+      iex> Scholar.Metrics.onehot_accuracy(y_true, y_pred)
+      #Nx.Tensor<
+        f32
+        0.5
+      >
 
   """
-  defn accuracy(y_true, y_pred) do
+  defn onehot_accuracy(y_true, y_pred) do
```

Comment:
I don't think we need one-hot versions of the functions, we can always convert one-hot to class numbers beforehand.